signer/src/error.rs

```diff
@@ -92,7 +92,12 @@ pub enum Error {
     #[error("{0}")]
     InvalidPrivateKey(#[source] secp256k1::Error),
 
-    /// This happens when we attempt to convert a [u8; 65] into a
+    /// This occurs when converting a byte slice to a [`PrivateKey`](crate::keys::PrivateKey)
```

Comment:
I would lop off the `(crate::keys::PrivateKey)` at the end here. VS Code can intellisense the [`PrivateKey`] type without it but cannot with it there.

Reply:
rustdoc can't infer the type if it's not in-scope (`use`'d), and adding a `use crate::keys::PrivateKey` to this file would result in an unused-import warning.

Reply:
👍🏽 yup makes sense.

---
Hmmm, I just rechecked my suggested change and it doesn't work anymore. I probably checked it incorrectly the first time. Now my VS Code intellisense picks up what you have here, and not my suggested change. Also, I have a lot of rustdoc warnings to clean up 😅.